assets/javascripts/discourse/templates/docs-index.hbs

```diff
@@ -1,6 +1,9 @@
 {{#conditional-loading-spinner condition=isLoading}}
-  {{#if emptyTopics}}
-    <span class="no-topics-found">{{html-safe (i18n "docs.no_topics")}}</span>
+  {{#if noContent}}
+    <EmptyState
```

Comment:
I believe we need to add a `.discourse-compatibility` file here, just to be safe with previous stable versions.

Reply:
Fixed it like this:

`
2.8.0.beta9: 05678c451caf2ceb192501da91cf0d24ea44c8e8
`

We could actually do 

`
2.8.0.beta8: 05678c451caf2ceb192501da91cf0d24ea44c8e8
`

since the EmptyState component was introduced one beta version earlier.  But then one version later we did [this improvement](https://github.com/discourse/discourse/commit/f97c8ff0a64a7bd15da87067183a1f9ed8402e4f) to the component to support styling in Teams. Using 2.8.0.beta9 in `.discourse-compatibility` would break styling of empty pages in Teams.